src/index/EncodedIriManager.h

```diff
@@ -206,7 +221,6 @@ class EncodedIriManagerImpl {
   // Equality operator for use in `TestIndexConfig`.
   QL_DEFINE_DEFAULTED_EQUALITY_OPERATOR_LOCAL(EncodedIriManagerImpl, prefixes_)
 
- private:
```

Comment:
Undo this, I think we can do everything with the extended interface from above now.